server/channels/jobs/helper_test.go

```diff
@@ -221,7 +237,11 @@ func (th *TestHelper) TearDown() {
 	}
 	th.ShutdownApp()
 	if th.tempWorkspace != "" {
-		os.RemoveAll(th.tempWorkspace)
+		err := os.RemoveAll(th.tempWorkspace)
+		if err != nil {
+			// Log error but don't fail tests
+			th.TestLogger.Error("Error removing temporary workspace", mlog.Err(err))
```

Comment:
I think the test should fail loudly here. The cleanup should work and if it doesn't we need to know.

Reply:
This is still a concern